sql/core/src/test/resources/sql-tests/results/describe-part-after-analyze.sql.out

```diff
@@ -93,7 +93,7 @@ Partition Values    	[ds=2017-08-01, hr=10]
 Location [not included in comparison]sql/core/spark-warehouse/t/ds=2017-08-01/hr=10	                    
 Created Time [not included in comparison]
 Last Access [not included in comparison]
-Partition Statistics	1121 bytes, 3 rows  	                    
+Partition Statistics	1229 bytes, 3 rows  	                    
```

Comment:
This is caused by adding `org.apache.spark.sql.create.version = 3.0.0-SNAPSHOT`?

Reply:
Right, @gatorsmile .

Reply:
Hm, does it mean that basically the tests will be failed or fixed for official releases (since it doesn't have `-SNAPSHOT`)?

Reply:
Hmmm .. yea, I think we should avoid ..

Reply:
It's filed and I made [a PR for SPARK-25971](https://github.com/apache/spark/pull/22972) for SQLQueryTestSuite.